client/src/pages/Home.js

```diff
@@ -1,12 +1,9 @@
 import { useEffect, useState } from "react";
 import { Link } from "react-router-dom";
-
 import "./Home.css";
-import logo from "./logo.svg";
-
+import PicCarousel from "../components/carousel";
```

Comment:
Maybe the fileName `carousel` should be the same as `PicCarousel`

Reply:
I agree with the suggested naming convention.